### ISSUE TITLE
v18.0.0 Missing theme files 

### ISSUE THREAD

**Issue by k3nsei** [dhutaryan/ngx-mat-timepicker#174]
![image](https://github.com/user-attachments/assets/929fca7b-2bb2-4a59-b1cd-f979f3269318)

![image](https://github.com/user-attachments/assets/b82ac8d8-4d7e-4acf-96d7-827336fed48e)

![image](https://github.com/user-attachments/assets/d5ec0d46-fa15-4f95-a227-b90882fc9843)

```bash
X [ERROR] Can't find stylesheet to import.
  ╷
3 │ @use './themes/base.scss';
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  node_modules\@dhutaryan\ngx-mat-timepicker\_timepicker-theme.scss 3:1  @forward
  node_modules\@dhutaryan\ngx-mat-timepicker\_index.scss 1:1             @use
```

**Comment by k3nsei** [comment]
@dhutaryan thanks for quick response

**Comment by dhutaryan** [comment]
Sorry, I've totally forgot to include new files into build. Fixed in `18.0.1`

**Comment by k3nsei** [comment]
It's totally ok, easy mistake to do. As including assets it's not common thing to do.
@dhutaryan Don't forget to mark previous package version as deprecated, because without those assets present it's unusable.

![image](https://github.com/user-attachments/assets/3752f325-d56b-47cb-b621-12b25bc69a64)

https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions
